src/hid.rs

```diff
@@ -0,0 +1,200 @@
++// Copyright � 2015, Alex Daniel Jones
++// Licensed under the MIT License <LICENSE.md>
++//! Taken from hidclass.h, hidsdi.h and hidpi.h
```

Comment:
Each header needs to have a corresponding module. So `hidclass.rs` `hidsdi.rs` and `hidpi.h`